### DISCUSSION TITLE
one div with aspect-square using multiple columns in column-3 div

### DISCUSSION THREAD

**Discussion by CodyMoltar** [tailwindlabs/tailwindcss#12286] [Help] [answered]
**What version of Tailwind CSS are you using?**

3.3.5

**What build tool (or framework if it abstracts the build tool) are you using?**

Sveltekit (1.2.0)

**What version of Node.js are you using?**

v20.9.0

**What browser are you using?**

Chrome 118.0.5993.117

**What operating system are you using?**

Windows 11

**Reproduction URL**

https://play.tailwindcss.com/8710nq4A51

https://svelte.dev/repl/c78a90ba16b440ae9cb1bc7785b3d297?version=4.2.2
(update the 'text' to trigger the output)

**Describe your issue**

When adding a div with an aspect-square class, the div uses more than just one column, and divides itself between them.

**Comment by adamwathan** [comment, marked as answer]
Hey! Yeah this is just what CSS does, nothing Tailwind-specific happening here:

https://jsfiddle.net/zkjp0e25/1/

So you'll probably just have to find some other way to do what you're trying to do.
